newspaper/network.py

```diff
@@ -119,8 +119,8 @@ def is_binary_url(url: str) -> bool:
         chars = len(
             [
                 char
-                for char in content
-                if 31 < ord(char) < 128 or ord(char) in [9, 10, 13]
+                for char in [ord(c) if isinstance(c, str) else c for c in content]
```

Comment:
why the precaution? we ensured at line 105 that content is not bytes
In my opinion it makes the code less readable like this. Am i missing something?

Reply:
yep, i see, the mistake is that decode with errors=replace should never raise an error. 
I will fix that in the merge. I think then it's safe to leave the ord(c) without any checking 
Thanks for pointing that out 👍